app/api/diagnostic/route.ts

```
@@ -340,152 +401,164 @@
-          anonymousSessionId: newAnonymousId // Send back for anonymous users
+          anonymousSessionId: newAnonymousId, // Send back for anonymous users
         });
 
-      case 'answer':
-        if (typeof sessionId !== 'string' || !sessionId) {
-          return NextResponse.json({ error: 'Invalid session ID' }, { status: 400 });
+      case "answer":
+        if (typeof sessionId !== "string" || !sessionId) {
+          return NextResponse.json({ error: "Invalid session ID" }, { status: 400 });
         }
         const validatedAnswer = validateAnswerRequest(data); // questionId is UUID of snapshotted q
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)